src/colmap/sfm/incremental_mapper.h

```diff
@@ -157,6 +159,7 @@ class IncrementalMapper {
   // pairs should be passed to `RegisterInitialImagePair`. This function
   // automatically ignores image pairs that failed to register previously.
   bool FindInitialImagePair(const Options& options,
+                            TwoViewGeometry& two_view_geometry,
                             image_t* image_id1,
```

Comment:
Let's change these from pointers to refs?

Reply:
Done, PTAL.